src/scales/ewbank.ts

```diff
@@ -26,17 +26,18 @@ const EwbankScale: GradeScale = {
     return getScore(grade)
   },
   getGrade: (score: number | Tuple): string => {
-    const validateScore = (score: number): number => {
+    const validateScore = (score: number): string => {
       const validScore = Number.isInteger(score) ? score : Math.ceil(score)
-      return Math.min(Math.max(0, validScore), routes.length - 1)
+      const index = Math.min(Math.max(0, validScore), routes.length - 1)
+      return routes[index].ewbank
     }
 
     if (typeof score === 'number') {
-      return routes[validateScore(score)].ewbank
+      return validateScore(score)
     }
 
-    const low: string = routes[validateScore(score[0])].ewbank
-    const high: string = routes[validateScore(score[1])].ewbank
+    const low: string = validateScore(score[0])
+    const high: string = validateScore(score[1])
```

Comment:
One last question: could you say again why this change was necessary?  The tests run for me without it.

Reply:
Yes of course. If I don't use type assertion in validateScore I get a type error **"Type 'string | number' is not assignable to type 'string'. Type 'number' is not assignable to type 'string'"**. I really can't explain to you why that only happens in ewbank or why that happens at all...